src/templates/partials/lighttpd.hbs

```diff
@@ -1,7 +1,11 @@
 # {{output.header}}
 # {{{output.link}}}
+$SERVER["socket"] ==     ":80" { }
+$SERVER["socket"] == "[::]:80" { }
+
 {{#if form.hsts}}
-$SERVER["socket"] == ":80" {
+#server.modules += ("mod_redirect")
```

Comment:
If we require a module that's not included by default, mention it at the top.

Reply:
ok.  will move those comments to the top